.travis.yml

```diff
@@ -36,31 +34,31 @@ before_install:
   - alias make="colormake -j2 "
 
   # Also setup a virtual display for after_success target when gui == true
-  - if ! $gui; then qbtconf="$qbtconf --disable-gui"; else export "DISPLAY=:99.0" && /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 ; fi
+  - if ! $gui; then qbtconf="$qbtconf --disable-gui" ; else export "DISPLAY=:99.0" && /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 ; fi
   - qbtconf="$qbtconf --with-qt4"
   - ltconf=" --with-libgeoip=system"
 
-  - echo settings
-  - echo $lt_source
-  - echo $ltconf
+  # Options for different branches
+  - if [[ "$lt_branch" == "RC_0_16" ]]; then qbtconf="$qbtconf --with-libtorrent-rasterbar0.16" ; fi
+
+  # Print settings
+  - echo $lt_branch
   - echo $gui
   - echo $qbtconf
+  - echo $ltconf
 
   - sudo apt-get -qq update
-  # Travis can stall during heavy load if these packages are installed in one step - split the command
+  # Travis may stall during heavy load if these packages are installed in one step - split the command
   - sudo apt-get -qq install debhelper autoconf automake colormake libssl-dev libgeoip-dev
-  # uncomment when Travis doesn't use Ubuntu 12.04 LTS, which has libtorrent 0.15.10 as package
-  #- sudo apt-get -qq install libboost-dev libboost-filesystem-dev libboost-system-dev
```

Comment:
I was thinking libtorrent is fetched from git/svn repo, so there is no need to rely on Ubuntu package any more.
And also, this line is mostly the same as the next line.

Anyway, sorry about this, I'll revert this if no further comments on this later.